src/main/java/org/folio/models/EncumbrancesProcessingHolder.java

```diff
@@ -63,11 +59,6 @@ public EncumbrancesProcessingHolder withEncumbrancesForRelease(List<Transaction>
     return this;
   }
 
-  public EncumbrancesProcessingHolder withEncumbrancesFromStorage(List<Transaction> encumbrancesFromStorage) {
```

Comment:
we deleted because it is not used any more?

Reply:
Right. And this is unrelated to the other changes.